src/renderer/components/attachment/messageAttachment.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { openAttachmentInShell } from '../message/messageFunctions';
+import { C } from 'deltachat-node/constants.enum';
+import { ScreenContext } from '../../contexts';
+import { isDisplayableByFullscreenMedia, isImage, isVideo, isAudio, getExtension, dragAttachmentOut, attachment } from './Attachment';
+
+const MINIMUM_IMG_HEIGHT = 150
+const MAXIMUM_IMG_HEIGHT = 300
+
+type AttachmentProps = { // TODO: replace "any" by the right type here
```

Comment:
Do you want to fix the TODO in the PR?

Reply:
No I believe typing the message type is out of scope for this pr, as its already large and the message type is probably our biggest type.

Reply:
Ok